Makefile

```diff
@@ -265,7 +265,12 @@ controller-gen: ## Download controller-gen locally if necessary.
 
 KUSTOMIZE = $(shell pwd)/bin/kustomize
 kustomize: ## Download kustomize locally if necessary.
-	$(call go-get-tool,$(KUSTOMIZE),sigs.k8s.io/kustomize/kustomize/v4@v4.0.5)
+	mkdir -p $$(pwd)/bin
```

Comment:
What's the point of these changes? Instead of single function that downloads any of the tools, we have to create separate process for each?

Reply:
The purpose is that go-get-tools fails to build kustomize 4.4.0 so I've changed it so that we download a binary artefact instead. That is all clearly explained in this [comment](https://github.com/k8ssandra/k8ssandra-operator/pull/286#issuecomment-1014115092), please let me know if you have quesitons.

Reply:
Works fine on my machines at least and cass-operator uses that method to download 4.4.1 for every integration test, so it works on GHA also. What in it doesn't work?

Reply:
I was getting an error when go-get-tools tried to build kustomize from source, I resolved it back on the [17th](https://datastax.slack.com/archives/C02023LQC1E/p1642385392477800) and the full [logs](https://github.com/k8ssandra/k8ssandra-operator/runs/4835404880?check_suite_focus=true) from those test runs are now deleted. The two fragments of the error I'd copied into involved  `undefined: os.ReadDir` and `note: module requires Go 1.16`.

Reply:
Are you trying to build with something older than go 1.16?

Reply:
Because both of those errors look like it, os.ReadDir was introduced in 1.16 and that module would require go 1.16 it seems.

Reply:
No, this error comes up when you try to upgrade kustomize to 4.4.0 and up. It doesn't happen locally, I only saw it on GHA, but it was consistently occurring.

Try installing kustomize 4.4.0 using go-get-tools on GHA and you'll see what I'm talking about. I'd be curious if you have any thoughts as to what's going on there, because it didn't make sense to me.

Reply:
Whatever version of Go I was trying to build with was simply the same one that GHA always uses (which I assume is a 1.16).

Reply:
As I said, this same method installs kustomize 4.4.1 in every GHA run on the cass-operator side and it works fine. And that method works fine in our other tests also. The reason you see that error is because there's wrong go version installed in the kuttl_tests.

Reply:
Michael, if I include this step, do you think the go-get-tools install will work? 

```
      - name: Set up Go 1.16
        uses: actions/setup-go@v1
        with:
          go-version: 1.16
      - uses: actions/cache@v2
```

Reply:
See PR #341